src/utils/__tests__/parseError.test.js

```diff
@@ -139,6 +139,40 @@ describe('parseError', () => {
           })
         })
 
+        describe('with a path and error array within errors', () => {
+          test('it parses the error correclty', () => {
+            const consoleMock = vi.spyOn(console, 'log')
+
+            const response = parseError({
+              response: {
+                data: {
+                  errors: [{
+                    path: ['Platforms', 0, 'ComposedOf'],
+                    errors: [
+                      'Composed Of must be unique. This contains duplicates named [ATM].'
+                    ]
+                  }]
+                },
+                status: 422
+              },
+              name: 'CMR Error'
+            })
+
+            expect(consoleMock).toBeCalledTimes(1)
+            // eslint-disable-next-line quotes
```

Comment:
You don't need this eslint-disable. You just need to change the ` to single quotes